tests/restapi/test_restapi.py

```diff
@@ -0,0 +1,138 @@
+import pytest
+import time
+import logging
+import requests
+import json
+
+from tests.common.helpers.assertions import pytest_assert
+from restapi_operations import Restapi
+
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('t0'),
+    pytest.mark.disable_loganalyzer
+]
+
+CLIENT_CERT = 'restapiclient.crt'
+CLIENT_KEY = 'restapiclient.key'
+
+restapi = Restapi(CLIENT_CERT, CLIENT_KEY)
+
+# Tests
+def test_data_path(construct_url):
+    params = '{"ip_addr": "10.3.152.32"}'
+    r = restapi.post_config_tunnel_decap_tunnel_type(construct_url, 'vxlan', params)
+    pytest_assert(r.status_code == 204)
```

Comment:
Can you break down this function into multiple testcases? parametrize should be good. The problem here is if one fails, then you don't test the rest. I am assuming all these cases are independent of each other

Reply:
Can you add a one line summary before each case so that it is easy to figure out failures from the logs?